examples/language_id_local_media_file.js

```diff
@@ -0,0 +1,50 @@
+const revai = require('revai-node-sdk');
+const token = require('./config/config.json').access_token;
+
+(async () => {
+    // Initialize your client with your Rev AI access token
+    var client = new revai.LanguageIdClient(token);
+
+    const jobOptions = {
+        metadata: 'node example language id local file submission',
+        notification_config: { url: 'https://jsonplaceholder.typicode.com/posts' },
+        delete_after_seconds: 30 * 24 * 60 * 60 // 30 days in seconds
+    };
+
+    // Media may be submitted from a local file
+    var job;
+    try {
+        job = await client.submitJobLocalFile('./resources/example.mp3',
+            jobOptions);
+    } catch (e) {
+        console.dir(e);
+    }
+
+    console.log('Language id job submitted.');
+    console.log(`Job Id: ${job.id}`);
+    console.log(`Status: ${job.status}`);
+    console.log(`Created On: ${job.created_on}`);
+
+    /**
+     * Waits 5 seconds between each status check to see if job is complete.
+     * note: polling for job status is not recommended in a non-testing environment.
+     * Use the notification_config option (see: https://docs.rev.ai/sdk/node/)
```

Comment:
@kbridbur is there a reason we dont link directly to the section 
https://docs.rev.ai/sdk/node/code-samples/#send-email-notifications-using-a-webhook

is it because that url could change/?